health-services/project-factory/migration/main/V20260813100000__add_attendance_sync_columns_campaign_data.sql

```
@@ -0,0 +1,19 @@
+ALTER TABLE eg_cm_campaign_data
+    ADD COLUMN IF NOT EXISTS isDeleted BOOLEAN DEFAULT false;
+
+-- De-enrolment carries an effective date, so a future-dated removal is still active today. Stored
+-- rather than applied on arrival, because the attendance event fires once and never again when the
+-- date comes round — downloads compare it against the current date instead.
+ALTER TABLE eg_cm_campaign_data
+    ADD COLUMN IF NOT EXISTS denrollmentDate BIGINT;
+
+-- Both attendance events identify their row by (type, uniqueIdAfterProcess) with no campaign scope.
+CREATE INDEX IF NOT EXISTS idx_eg_cm_campaign_data_unique_id_after_process
+    ON eg_cm_campaign_data (type, uniqueIdAfterProcess);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)